packages/react-button/src/components/Button/Button.types.ts

```diff
@@ -21,30 +21,30 @@ export type ButtonProps = ComponentProps &
     //  */
     // loader?: ShorthandProps<React.HTMLAttributes<HTMLSpanElement>>;
 
-    // /**
-    //  * A button can fill the width of its container.
-    //  * @default false
-    //  */
-    // block?: boolean;
+    /**
+     * A button can fill the width of its container.
+     * @default false
+     */
+    block?: boolean;
 
-    // /**
-    //  * A button can have completely rounded corners.
-    //  * @default false
-    //  */
-    // circular?: boolean;
+    /**
+     * A button can have completely rounded corners.
+     * @default false
+     */
+    circular?: boolean;
 
     /**
      * A button can show that it cannot be interacted with.
      * @default false
      */
     disabled?: boolean;
 
-    // /**
-    //  * When set, allows the button to be focusable even when it has been disabled. This is used in scenarios where it
-    //  * is important to keep a consistent tab order for screen reader and keyboard users.
-    //  * @default false
-    //  */
-    // disabledFocusable?: boolean;
+    /**
+     * When set, allows the button to be focusable even when it has been disabled. This is used in scenarios where it
+     * is important to keep a consistent tab order for screen reader and keyboard users.
```

Comment:
Minor, minor thing that doesn't need to be changed within this PR: I'd suggest specifying that this is primarily used in scenarios where the disabled button is within a menu or commandbar, and not for standalone buttons.

Reply:
I specified those scenarios in the comment.